torch/nn/quantized/modules/linear.py

```diff
@@ -61,3 +61,17 @@ def forward(self, x):
             self.out_scale,
             self.out_zero_point)
         return Y_q
+
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super()._save_to_state_dict(destination, prefix, keep_vars)
```

Comment:
given that the state_dict for linear is literally bias+weight, I don't think it makes sense to call parent method - you can just add 'bias' handling here

Reply:
there are 'output_scale' and 'output_zero_point' as well